microsoft-outlook-365/1.0/modules/ROOT/pages/microsoft-outlook-365-connector-reference.adoc

```diff
@@ -2101,6 +2101,7 @@ Updates the properties of a message object.
 == On Modified Event
 `<outlook365:modified-event-listener>`
 
+Triggers a flow for modified events.
```

Comment:
I think this wording is better. Regarding trigger vs initiate - they can be interchangeable but because some of the docs call sources trigger, I avoid the term in descriptions - up to you whether to use it or not.

Initiates a flow when an event is modified.

Reply:
I like this a lot better!